ui/app/models/database/connection.js

```diff
@@ -147,14 +199,19 @@ export default Model.extend({
     defaultShown: 'Default',
   }),
 
+  // ENTERPRISE ONLY
+  self_managed: attr('boolean', {
+    subText:
+      'If set, allows onboarding static roles with a rootless connection configuration. Mutually exclusive with username and password. If set, will force verify_connection to be false.',
```

Comment:
Is a future improvement, it might be good to have this field automatically toggle `verify_connection` but all of the databases use the same form, so this is a larger refactor.

Reply:
"If set," is used twice and thus reads a little funny. You could remove the first usage.

Reply:
Thanks! Just copied this directly from the API docs 🙃